dist/index.js

```diff
@@ -4928,18 +4931,24 @@ async function getRebar3Versions() {
 function getVersionFromSpec(spec, versions) {
   let version = null
 
-  if (core.getInput('version-type', { required: false }) === 'strict') {
+  if (
+    spec.match(/rc/) ||
```

Comment:
Since we're not allowing `rc` below we try to fetch one from the listing, here. If it is an `rc` we consider the match to be strict.